### ISSUE TITLE
simplify the whole projet

### ISSUE THREAD

**Issue by fmigneault** [crim-ca/weaver#5]
remove supervisor nginx mongodb // only run pyramid app

will save a LOT of space for image size
will simplify by a LOT the image/config setup

## todo

- remove buildout and related mongodb, supervisor, etc.. parts
- use directly gunicorn instance in docker
- link to external mongodb
- remove installs in following make according to removed dependencies
```
Makefile:181
"$(CONDA_HOME)/bin/conda" install -y -n "$(CONDA_ENV)" "setuptools=$(SETUPTOOLS_VERSION)" supervisor nginx mongodb
```
- figure out how to handle celery... external one would duplicate code a lot...
- figure out how to serve pywps endpoint, pywps-data-output dir endpoint and weaver app all under the same pyramid app (maybe using `[composite:main]` in `weaver.ini` ?)
- update compose of other repos to update `weaver.ini` file directly instead of `custom.cfg` since it won't be used anymore + add link / other images required

**Comment by davidcaron** [comment]
Couldn't agree more. That will help development and deployment a lot.

**Comment by davidcaron** [comment]
FYI bird-house/twitcher doesn't use buildout anymore.

**Comment by fmigneault** [comment]
fixed by #59
